tests/test_annotations.py

```diff
@@ -398,3 +398,16 @@ def test_popup(caplog):
     target = "annotated-pdf-popup.pdf"
     writer.write(target)
     Path(target).unlink()  # comment this out for manual inspection
+
+def test_outline_action_without_d_lenient(pdf_file_path):
+    pdf_path = RESOURCE_ROOT / "issue-3268.pdf"
```

Comment:
We prefer to keep the repository size small if possible. Please consider either downloading it on the fly or provide it at https://github.com/py-pdf/sample-files under the terms of the CC-BY-SA-4.0 if you have the necessary rights to do so.